pydis_site/apps/api/admin.py

```diff
@@ -3,7 +3,8 @@
 from .models import (
     BotSetting, DeletedMessage,
     DocumentationLink, Infraction,
-    MessageDeletionContext, OffTopicChannelName,
+    MessageDeletionContext, Nomination,
+    OffTopicChannelName,
```

Comment:
if you already change the number of imports per line, this should probably just be fixed to one symbol per line

Reply:
Yes, good point. Didn't really think properly about it.

I think that keeping it at two per line is probably better, due to the sheer number of imports we do here. It's not like it's unreadable with two symbols per line here, it's not a complicated file or import structure. I'll clean it up and reformat it to the old style of two per line.